packages/components/src/resizable-box/stories/index.tsx

```diff
@@ -0,0 +1,94 @@
+/**
+ * External dependencies
+ */
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+
+/**
+ * Internal dependencies
+ */
+import ResizableBox from '..';
+
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+const meta: ComponentMeta< typeof ResizableBox > = {
+	title: 'Components/ResizableBox',
+	component: ResizableBox,
+	argTypes: {
+		__experimentalShowTooltip: { control: 'boolean' },
+		minHeight: { control: 'number' },
+		minWidth: { control: 'number' },
+		onResizeStop: { action: 'onResizeStop' },
+		showHandle: { control: 'boolean' },
```

Comment:
Do you know why are these props not shown by default in the Storybook docs?

Reply:
Good question. I tweaked the export so at least our first-party types are visible ✅ 

However, all the props coming from the upstream [`re-resizable`](https://github.com/WordPress/gutenberg/blob/d3eaa6cacb4038fd2cab7f49ef4444e64ba6e6eb/packages/components/src/resizable-box/index.tsx#L90) are not shown at all 🤔 So I think what will happen is that in the full TypeScript migration, we'll decide which ones of the upstream props are noteworthy, and add our own typings/descriptions for them so they'll be shown in the Storybook props table. For all the other upstream props, the main component description will nudge people toward the `re-resizable` docs as written in the [readme](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/resizable-box/README.md).

Reply:
Sounds like a plan